### PR TITLE
Add small icon versions of the logo and a cleaned up svg (without overla...

### PR DESCRIPTION
**Pull request by cillianderoiste**
...ps)

### CODE REVIEW

File: logo/nix-snowflake.svg [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Layer 1"
+     inkscape:groupmode="layer"
+     id="layer1">
+    <g
+       id="g3773"
+       transform="translate(-62.857169,-117.14286)"
+       inkscape:export-filename="/home/goibhniu/NixFonts/snowflake.png"
```
Comment (edolstra) [inline logo/nix-snowflake.svg]: Should these filenames be included?

Comment (cillianderoiste) [inline logo/nix-snowflake.svg]: They're the paths to where I have previously exported parts of the file. They don't do any harm, but there is an option to export an "Optimized SVG" which strips these and other metadata. I'll give that a shot and update the PR next week. Thanks for checking!

**Comment by cillianderoiste** [comment]
I've updated the SVG version to strip unnecessary Inkscape metadata using the Optimized SVG export option.

**Comment by cillianderoiste** [comment]
@edolstra is this OK now?

**Pull request merged**